src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/OrchestratorNotificationTask.java

```diff
@@ -79,6 +70,21 @@ public void run() {
         log.info("Finished {} job", TASK_NAME);
     }
 
+    @Transactional
+    private void processEnvelope(AtomicInteger successCount, Envelope env) {
+        try {
+            serviceBusHelper.sendMessage(new EnvelopeMsg(env));
```

Comment:
this will not change the logic. we always try to sent to bus first and then insert to event.